src/test/java/no/nav/familie/integrasjoner/arbeidsfordeling/ArbeidsfordelingTestConfig.kt

```diff
@@ -0,0 +1,35 @@
+package no.nav.familie.integrasjoner.arbeidsfordeling
+
+import no.nav.tjeneste.virksomhet.arbeidsfordeling.v1.binding.ArbeidsfordelingV1
+import no.nav.tjeneste.virksomhet.arbeidsfordeling.v1.binding.FinnBehandlendeEnhetListeUgyldigInput
+import no.nav.tjeneste.virksomhet.arbeidsfordeling.v1.informasjon.Organisasjonsenhet
+import no.nav.tjeneste.virksomhet.arbeidsfordeling.v1.meldinger.FinnBehandlendeEnhetListeRequest
+import no.nav.tjeneste.virksomhet.arbeidsfordeling.v1.meldinger.FinnBehandlendeEnhetListeResponse
+import org.mockito.ArgumentMatchers
+import org.mockito.Mockito
+import org.springframework.context.annotation.Bean
+import org.springframework.context.annotation.Configuration
+import org.springframework.context.annotation.Primary
+import org.springframework.context.annotation.Profile
+
+@Configuration
+class ArbeidsfordelingTestConfig {
+
+    @Bean
+    @Profile("mock-arbeidsfordeling")
+    @Primary
+    @Throws(FinnBehandlendeEnhetListeUgyldigInput::class)
+    fun arbeidsfordelingMock(): ArbeidsfordelingV1? {
+        val arbeidsfordelingV1 = Mockito.mock(ArbeidsfordelingV1::class.java)
```

Comment:
Kan du bruke `mockk` i denne klassen også? Syns det er best å ikke blande flere rammeverk for mocking i samme kodebase :)